libqtile/widget/systray.py

```diff
@@ -40,15 +40,16 @@
 
 class Icon(window._Window):
     _windowMask = EventMask.StructureNotify | \
+        EventMask.PropertyChange | \
         EventMask.Exposure
 
     def __init__(self, win, qtile, systray):
         window._Window.__init__(self, win, qtile)
         self.systray = systray
-        self.width = systray.icon_size
-        self.height = systray.icon_size
+        self.protocol_version = 1
```

Comment:
I'm just checking the XEmbed spec, should this be 0?

from http://standards.freedesktop.org/xembed-spec/xembed-spec-latest.html#lifecycle